app/common/filters/time_and_date_filters.py

```diff
@@ -0,0 +1,7 @@
+from datetime import datetime
+
+from babel import dates
+
+
+def format_date(value: datetime) -> str:
```

Comment:
It would be good at some point to have the various formats in the design system so that we don't have to keep adding or changing formats as we go - maybe we can make that a small BAU story to add them and give them good descriptive names https://www.gov.uk/guidance/style-guide/a-to-z#dates

Reply:
Ticket created here https://mhclgdigital.atlassian.net/browse/FSPT-405